main/tests/test_oa_hash_map.cpp

```diff
@@ -210,6 +210,65 @@ MainLoop *test() {
 		}
 	}
 
+	// Test map with 0 capacity.
+	{
+		OAHashMap<int, String> original_map(0);
+		original_map.set(1, "1");
+		OS::get_singleton()->print("OAHashMap 0 capacity test passed.\n");
```

Comment:
This test will only fail if it crashes due to the division by zero. I think we would be even in a better position if we checked that the capacity has actually been set to `1`.

As I say in the other comment, a public `get_capacity()` would be the most convenient way to do that check at the moment.

Reply:
See my above comment.